src/helpers.js

```diff
@@ -62,6 +67,70 @@ function readFileString (file) {
   })
 }
 
+/**
+ * Load local private key
```

Comment:
this is a public key. We have to be careful with naming, and for security review. Private keys are never ever downloaded